# DebugInfo.msg
Header header

# Arm Controller Info
float32 arm_target_pos_x
float32 arm_target_pos_y
float32 arm_target_pos_z
bool ik_solution_found

# Hand Retargeting Info
float32 total_loss
float32 loss_align
float32 loss_couple
float32 loss_smooth